APE24.rst

```diff
@@ -0,0 +1,507 @@
+  This way, the packages would be listed both over at pyOpenSci and at Astropy
+  websites.
+* An Astropy-specific page on the pyOpenSci website that would be a dedicated
+  link for only Affiliated packages. We could then include some other
+  information about the Astropy ecosystem and branding (logo, colors, etc).
+  A listed package would also have a link back to the review that was done.
+* A link back to Astropy website from pyOpenSci, with the understanding that
+  Astropy would do vice versa. This is similar to what pyOpenSci has done with Pangeo.
+
+
+Branches and pull requests
+==========================
+
+.. Any pull requests or development branches containing work on this APE should be
+.. linked to from here.  (An APE does not need to be implemented in a single pull
+.. request if it makes sense to implement it in discrete phases). If no code is yet
+.. implemented, just put "N/A"
+
+Issues:
+
+* `Affiliated Package: Consolidate with pyOpenSci? <https://github.com/astropy/astropy-project/issues/334>`_
+* `Metrics for community partnership landing pages to track health <https://github.com/pyOpenSci/software-peer-review/issues/226>`_
+  (as noted above in **"Re-review" of packages**, this issue is not required
+  to be resolved for this APE to be accepted)
+
+Pull requests:
+
+* `Feat: add text associated with astropy partnership <https://github.com/pyOpenSci/software-peer-review/pull/225>`_
+* `FEAT: Start at community mockup / astropy <https://github.com/pyOpenSci/pyopensci.github.io/pull/207>`_
+* `Add: more specific community partners page with FAQ <https://github.com/pyOpenSci/software-peer-review/pull/253>`_
+* *TODO: A future PR by William Jamieson to update our Affiliated guidelines and process,
+  after consolidating what we have with that of pyOpenSci.*
+
+
+Implementation
+==============
+
+.. This section lists the major steps required to implement the APE.  Where
+.. possible, it should be noted where one step is dependent on another, and which
+.. steps may be optionally omitted.  Where it makes sense, each  step should
+.. include a link related pull requests as the implementation progresses.
+
+If we decide to move forward with this partnership, these are the proposed steps:
+
+1. Do a trial review period of some packages to see how it goes to make the
+   Astropy community feel more comfortable with the changes
+   (also see **The trial period** below).
+2. Based on the trial, our current Affiliated editors would finalize
+   and publish the updated guidelines and process, including new
+   COI policy, packaging guidelines, and so forth.
+   These new guidelines need to be compatible with
+   `pre-APE 24 Affiliated Package review guidelines`_ and
+   `pre-APE 24 Affiliated Package review process, with templates`_.
+   At the same time, we should link to more modern
+   practices over at pyOpenSci and `Scientific Python`_, in addition
+   to the "old ways" in Astropy and OpenAstronomy.
```

Comment:
```suggestion
   At the same time, we should link to the actively developed
   practices over at pyOpenSci and `Scientific Python`_, in addition
   to the already implemented ones in Astropy and OpenAstronomy.
```
I think this captures it better than "more modern" (although OpenAstronomy is also still being actively developed),
but perhaps also reminds us that not everything can be substituted at once (e.g. the pyOS guide does not have a section on testing yet).

Reply:
hey @pllim @dhomeier 

1. could we please link to our packaging guide here ??

_practices over at pyOpenSci
https://www.pyopensci.org/python-package-guide/_

2. I wonder if we want to be more explicit and say 
At the same time, we should link to the actively developed
   packaging guides developed by the pyOpenSci (link to our giude) and `Scientific Python`_, (link to scipy / henry's guide) communities in addition
   to the already implemented guidelines found in Astropy and OpenAstronomy (link to the open astronomy).

 I would argue that the practices in both our guide and henry's at scientific python are more modern leaning. I think henry is literally updating that guide when new peps and tools come out as he is so invested in and knowledgable about that space!!

However, for me i don't personally have a preference for modern vs maintained in this APE and welcome @dhomeier language!! it's good language and supports the important work those who developed the astro guide have done.   

i love the idea of adopting "newer" packaging approaches in our ecosystem slowly and steadily to support users. 

i hope that helps @pllim ping me again if it doesnt

Reply:
I'm not specifically opposed to any of the changes / links suggested here. However, I feel that we are wasting our time - this is an APE that will allow the Astropy community to formally accept or decline this change. It's not a living document. Once accepted, it's archived. Sure, we can go back and look at it, but it's not meant to be up-to-date, and it's not the doc the package authors will look at. 

So, my suggestion is to stop fine-tuning this and just close and accept it. Instead, we should spend our efforts in palces that people will actually look at when they propose and look up accepted pacakges.

Reply:
i will go ahead and approve on my end y 'all. @hamogu my apologies i didn't understand exactly how this APE will be used /viewed.

Reply:
I have added the requested changes/links to packaging guides above.

Admittedly I was not sure about the binding power of the APE either, but in light of the comments from @Cadair and others about describing our current process in an overly negative way, just wanted to avoid framing things here again in a "modern" vs. "outdated" or "obsolete" picture.